example/bench/recipe.cpp

```diff
@@ -0,0 +1,54 @@
+#include <random>
+
+#include <benchmark_cell.hpp>
+#include <common_types.hpp>
+#include <time_sequence.hpp>
+
+#include "recipe.hpp"
+
+cell_size_type bench_recipe::num_cells() const {
+    return params_.num_cells;
+}
+
+arb::util::unique_any bench_recipe::get_cell_description(cell_gid_type gid) const {
+    using RNG = std::mt19937_64;
+    arb::benchmark_cell cell;
+    cell.realtime_ratio = params_.cell.realtime_ratio;
+    cell.time_sequence = arb::poisson_time_seq<RNG>(RNG(gid), 0, 1e-3*params_.cell.spike_freq_hz);
```

Comment:
Comment on what's happening here? Are we seeding a sequence based on the gid?

Reply:
Yes, we are passing it a random generator seeded on `gid`. This way, regardless of how the cell is distributed over MPI ranks/threads/time zones, it will generate a reproducable sequence of spikes.
I have added a comment that explains this in more detail.